realtime_ai_character/main.py

```diff
@@ -28,9 +29,28 @@
 
 app.include_router(restful_router)
 app.include_router(websocket_router)
-app.mount("/static", StaticFiles(directory=os.path.join(
-    os.path.dirname(os.path.abspath(__file__)), 'static')), name="static")
 
+web_build_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 
+                              '..', 'client', 'web', 'build')
+
+if os.path.exists(web_build_path):
```

Comment:
in the else case, also add an endpoint to prompt user to run web_build or use react frontend?

Reply:
+1, but don't error out, just prompt a warning or something, because for our dev server and prod server, we are likely not going to need it